app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt

```diff
@@ -76,11 +75,12 @@ internal class MovieDetailsViewModelTest {
 
     @Test
     fun `when method get throwable from use case, errorSnackBar should run`() = runTest {
-        coEvery { getMovieDetailsUseCase(movieId) } returns badResponse
-        every { movieDetailsFragmentNavigator.errorSnackBar(errorMessage) } just Runs
+        val slot = slot<() -> Unit>()
+        coEvery { getMovieDetailsUseCase(movieId) } returns badResponse andThen Result.success(movieDetailsModel)
+        every { movieDetailsFragmentNavigator.errorSnackBar(errorMessage, onAction = capture(slot)) } answers { slot.captured.invoke() }
 
-        movieDetailsFragmentNavigator.errorSnackBar(errorMessage)
+        MovieDetailsViewModel(fragmentArgs.toSavedStateHandle(), getMovieDetailsUseCase, movieDetailsFragmentNavigator).getMovieDetails()
```

Comment:
Would be better to extract a `tested` variable here to improve the readability

Reply:
I extracted it to the variable